### PR TITLE
fix: update identity state for mtp credentials

### PR DESCRIPTION
**Pull request by martinsaporiti**

### CODE REVIEW

File: internal/core/services/identity.go [hunk 1]
```diff
@@ -310,6 +310,7 @@ func (i *identity) GetKeyIDFromAuthClaim(ctx context.Context, authClaim *domain.
 }
 
 func (i *identity) UpdateState(ctx context.Context, did w3c.DID) (*domain.IdentityState, error) {
+	const authBJJCredential = "https://schema.iden3.io/core/jsonld/auth.jsonld#AuthBJJCredential"
```
Comment (x1m3) [inline internal/core/services/identity.go:313]: I think that we could use domain.AuthBJJCredentialTypeID constant

**Review by x1m3 — COMMENTED** [review]
LGTM. Just left a comment regarding a constant that probably we could reuse

**Review by x1m3 — APPROVED** [review]

**Review by javip97 — APPROVED** [review]
LGTM

**Review by x1m3 — APPROVED** [review]
LGTM

**Pull request merged**